docs: mention import-spec upgrade (#31)
neo4j/docs-dataflow-connector: common-content-dataflow/modules/ROOT/partials/changelog.adoc

To check the release status of https://cloud.google.com/compute/docs/regions-zones[a particular region], you can run `gsutil ls gs://dataflow-templates-$\{region}/` (e.g. `gsutil ls gs://dataflow-templates-europe-west2/`).
It is not possible to run any version other than the *latest* one deployed in a given region.

== Release 2026-02-17-00_RC00

* Upgrade import-spec from 1.0.0-rc03 to 1.0.0-rc11

> This allows users to define mapping overrides for the key properties of start and end nodes

Reviewer: ```suggestion
Support for defining mapping overrides for the key properties of start and end nodes
```
